### DISCUSSION TITLE
Collection : multiple contains condition and word with accent

### DISCUSSION THREAD

**Discussion by mediactive** [statamic/cms#6937] [Q&A]
Hello,
I would like to allow keywords search in 2 fields of a collection (title and description).
I use this code:
{{ collection:public_notices sort="start_at:desc" description:contains="{ $keyword1 }" title:contains="{ $keyword1 }" description:contains="{ $keyword2 }" title:contains="{ $keyword2 }" }}
It doesn't work.
This code, with a single "contains" condition works:
{{ collection:public_notices sort="start_at:desc" description:contains="{ $keyword1 }" }}
How to allow the search with several keywords and display results containing at least 1 of the keywords used for the search.

How to do the "contains" condition work with accented words, like "québec"?

Thanks a lot for your help!

**Comment by jasonvarga** [comment]
The `contains` conditions map to `where like` clauses.

You can't do multiple like you've noticed but you could whip up a quick custom scope to handle it.

https://statamic.dev/extending/query-scopes-and-filters#scopes

```php
<?php

namespace App\Scopes;

use Statamic\Query\Scopes\Scope;

class Keywords extends Scope
{
    public function apply($query, $values)
    {
        $keywords = explode('|', $values['keywords']);

        $query->where(function ($query) use ($keywords) {
            foreach ($keywords as $keyword) {
                $query
                    ->orWhere('title', 'like', '%'.$keyword.'%')
                    ->orWhere('description', 'like', '%'.$keyword.'%');
            }
        });
    }
}
```

```antlers
{{ collection:articles filter="keywords" keywords="{keyword1}|{keyword2}" }}
```
